sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +290,27 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map(
+      "ID" -> "Execution ID of the SQL query.",
+      "Description" -> "The description links to the query details page",
+      "Submitted" -> "Submitted",
+      "Duration" -> "Difference between start time and close time.")
+
+    if (showRunningJobs && showSucceededJobs && showFailedJobs) {
+      headerNameWithTooltips += (
+        "Running Job IDs" -> "IDs of running jobs.",
```

Comment:
Likewise these don't add any info over the column name. "X ID"  is pretty much the same as "ID of X"

Reply:
@srowen, any suggestion to show tooltips msg (I am just showing the tooltips msg same as header name if tooltips msg not require to maintain consistency in UI)